activiti-api-process-model/src/main/java/org/activiti/api/process/model/payloads/TimerPayload.java

```diff
@@ -0,0 +1,245 @@
+package org.activiti.api.process.model.payloads;
+
+import java.util.Date;
+import java.util.UUID;
+
+import org.activiti.api.model.shared.Payload;
+
+public class TimerPayload implements Payload {
+
+    private String id;
+    
+    private Date dueDate;
+    private Date endDate;
+
+    private String executionId;
+    private boolean isExclusive;
+
+    private int retries;   
+    
+    private int maxIterations;
+    private String repeat;
+
+    protected String jobHandlerType;
+    protected String jobHandlerConfiguration;
+    
+    protected String exceptionMessage;
+
+    protected String tenantId;
```

Comment:
For sure we don't want to expose `tenantId`.

Reply:
Have removed some parameters: https://github.com/Activiti/activiti-api/pull/128/commits/4e0b51e7d3470e05c98933bfc3c9b5cda25d806d